build fail likely real
```
======================================================================
FAIL: test_repeater (corehq.apps.repeaters.tests.test_repeater.RepeaterTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/repeaters/tests/test_repeater.py", line 143, in test_repeater
    self.assertEqual(len(repeat_records), 2)
AssertionError: 3 != 2
----------------------------------------------------------------------
```

--- a/corehq/apps/repeaters/tests/test_repeater.py
+++ b/corehq/apps/repeaters/tests/test_repeater.py
@@ -246,22 +246,29 @@ class CaseRepeaterTest(BaseRepeaterTest, TestXmlMixin):
 class RepeaterFailureTest(BaseRepeaterTest):
 
     def setUp(self):
-        self.domain = "test-domain"
-        create_domain(self.domain)
+        self.domain_name = "test-domain"
+        self.domain = create_domain(self.domain_name)
 
         self.repeater = CaseRepeater(
-            domain=self.domain,
+            domain=self.domain_name,
             url='case-repeater-url',
             version=V1,
-            format='new_format'
+            format='other_format'
         )
         self.repeater.save()
         self.post_xml(xform_xml, self.domain)
 
+    def tearDown(self):
+        self.domain.delete()
+        self.repeater.delete()
+        repeat_records = RepeatRecord.all()
+        for repeat_record in repeat_records:
+            repeat_record.delete()
+
     def test_failure(self):
         payload = "some random case"
 
-        @RegisterGenerator(CaseRepeater, 'new_format', 'XML')
+        @RegisterGenerator(CaseRepeater, 'other_format', 'XML')
         class NewCaseGenerator(BasePayloadGenerator):
             def get_payload(self, repeat_record, payload_doc):
                 return payload

--- a/corehq/tests/app_test_db_dependencies.yml
+++ b/corehq/tests/app_test_db_dependencies.yml
@@ -165,3 +165,22 @@ corehq.form_processor:
 corehq.apps.toggle_ui: []
 corehq.doctypemigrations:
   - corehq.couchapps
+corehq.apps.repeaters:
+  - casexml.apps.case
+  - casexml.apps.stock
+  - corehq.apps.commtrack
+  - corehq.apps.locations
+  - corehq.apps.products
+  - corehq.apps.domain
+  - corehq.apps.tzmigration
+  - corehq.apps.users
+  - corehq.couchapps
+  - couchforms
+  - django_prbac
+  # unfortunately domain.delete() needs all of these - otherwise they could be removed
+  - custom.logistics
+  - custom.ilsgateway
+  - custom.ewsghana
+  - django_prbac
+  - corehq.apps.accounting
+  - corehq.apps.smsforms

